### ISSUE TITLE
have plans to create scripts or .py files that can run on Windows in the future?

### ISSUE THREAD

**Issue by c469591** [open-mmlab/Amphion#89]
Hello, I was wondering if there are any plans to create .py files that can run on Windows?
Currently, all the scripts in our project are .sh files, which cannot be executed on Windows.
I'm also considering whether to spend time converting the .sh scripts into .py. If someone is already working on this, then I'll just wait for it.
Thank you!

**Comment by RMSnow** [comment]
Hi @c469591, we have no plans to directly port to the Windows/MacOS platform until now, since it will take much labor and testing cost. Besides, most models of Amphion are dependent on big data and large-scale training, so we think Linux may be a better choice, at least for now. If a Linux-based server is not available, we recommend to use huggingface space or colab to play Amphion.

However, we are planning to integrate some typical models into Colab/Jupyter Notebook, just for teaching and education. The specific date has not been decided. We hope to release them in this summer :)

**Comment by c469591** [comment]
hi，
I understand,
I've also tried rewriting .sh files on Windows, but it's more difficult than I imagined. However, I will continue to try.
Additionally, based on my own experience, Google Colab doesn't seem very user-friendly; it limits the runtime for each session and is not very stable. It's not a good option for me.
I'll give Hugging Face a shot, but currently, I'm still testing on AutoDL.
The downside of using GPUs on AutoDL is that it constantly costs money which is quite troubling for me.
And since my computer with an NVIDIA 3060 runs on Windows system—the virtual Linux subsystem in Windows isn't fully functional—it really gives me a headache.
